clustered/server/src/main/java/org/ehcache/clustered/server/internal/messages/EhcacheMessageTrackerMessage.java

```diff
@@ -31,8 +31,6 @@
  */
 public class EhcacheMessageTrackerMessage extends EhcacheSyncMessage {
 
-  public static final int UNKNOWN_SEGMENT = -1;
-
   private final int segmentId;
```

Comment:
Even this `segmentId` can be removed from the message and be computed on the passive from the concurrency key in `InvokeContext`.

Reply:
Let's do that in a different PR since this is simply a revert of a given commit if that's ok with you.